aea/cli/registry/push.py

```diff
@@ -93,3 +95,59 @@ def push_item(item_type: str, item_name: str) -> None:
             item_type, item_name, resp['public_id']
         )
     )
+
+
+def get_packages_path() -> str:
+    """
+    Get path to packages folder.
+
+    Should not be called from outside the project dir.
+
+    :return: str path to packages dir.
+    """
+    project_parent_dir = os.path.abspath('__file__').split(
```

Comment:
We have a path already defined as a constant: `DEFAULT_REGISTRY_PATH`. Use this instead.

Reply:
I've tried. But it always leads to "packages" one level up. It won't work with any nested folder structure.
Sure I can use that. But don't you think it's better to have a possibility to use it with nested structure?

Reply:
well you neet to construct the correct path I suppose. `packages` is assumed to be on the root dir. So if you build cwd + DEFAULT_REGISTRY+PATH then cwd would have to be the agent project dir. You can figure out whether you're in agent project dir by checking if the aea-config file is present.